tests/tests.js

```diff
@@ -81,23 +74,23 @@ define(['jquery', 'zimArchive', 'zimDirEntry', 'util', 'utf8'],
      * @param e Error
      */
     function errorHandlerAsyncTest(e) {
-        ok(false, "Error in async call", e);
-        start();
+        assert.ok(false, "Error in async call", e);
+        done();
     }
```

Comment:
Okay I now see the QUnit no try-catch option which was causing all the errors to be swallowed up. Should be okay to remove this.